src/main/java/success/planfit/controller/TimetableController.java

```diff
@@ -0,0 +1,55 @@
+package success.planfit.controller;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import success.planfit.controller.utils.ControllerUtil;
+import success.planfit.dto.request.TimetableCreationRequestDto;
+import success.planfit.dto.request.TimetableUpdateRequestDto;
+import success.planfit.dto.response.TimetableInfoResponseDto;
+import success.planfit.service.TimetableService;
+
+
+import java.security.Principal;
+import java.time.LocalDate;
+import java.util.List;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("/timetable")
+public class TimetableController {
+
+    private final ControllerUtil controllerUtil;
+    private final TimetableService timetableService;
+
+
+    // 날짜를 기준으로 시간표 등록
+    @PostMapping("/add/{date}")
+    public ResponseEntity<Void> addTimetable(@PathVariable LocalDate date, @RequestBody TimetableCreationRequestDto requestDto, Principal principal) {
```

Comment:
```suggestion
    // 날짜를 기준으로 시간표 등록
    @PostMapping("/{date}")
    public ResponseEntity<Void> addTimetable(@PathVariable LocalDate date, @RequestBody TimetableCreationRequestDto requestDto, Principal principal) {
```
`TimetableController`와 `UserController`에 `add`, `remove`, `update`, `get`처럼 컨트롤 URI를 사용하셨네요.
HTTP 메서드를 통해 동작을 표현하지 않고 URI에 동사를 사용한 이유가 따로 있을까요?
현재의 URI가 적합한지, HTTP 메서드만으로 동작을 표현하는 것이 적합한지 의논해 보고 싶습니다.
이해가 어렵다면 아래 벨로그를 참고하면 좋을 것 같습니다.

[참고 문서](https://velog.io/@da_na/HTTP-API-URI-%EC%84%A4%EA%B3%84-%ED%94%84%EB%A1%9C%EC%A0%9D%ED%8A%B8%EC%97%90-%EC%BB%A8%ED%8A%B8%EB%A1%A4-URI-%EC%84%A4%EA%B3%84%ED%95%98%EA%B8%B0)